### PR TITLE
Add support for Kubernetes 1.12.4

### PR DESCRIPTION
**Pull request by mboersma**
**What this PR does / why we need it**:

See https://github.com/kubernetes/kubernetes/releases/tag/v1.12.4

Refs #73.

TODO:

- [x] upload Windows binary

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:
```release-note
Add support for Kubernetes 1.12.4
```

### CODE REVIEW

File: pkg/api/k8s_versions.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ var k8sComponentVersions = map[string]map[string]string{
 		"aci-connector":                    "virtual-kubelet:latest",
 		ContainerMonitoringAddonName:       "oms:ciprod10162018-2",
 		AzureCNINetworkMonitoringAddonName: "networkmonitor:v0.0.4",
-		"cluster-autoscaler":               "cluster-autoscaler:v1.3.3",
+		"cluster-autoscaler":               "cluster-autoscaler:v1.12.1",
```
Comment (jackfrancis) [inline pkg/api/k8s_versions.go]: Is there a good reason for including this change in this PR?

Comment (mboersma) [reply pkg/api/k8s_versions.go]: That seems incorrect then. Should be 1.12.x for 1.12.x and 1.13.x for 1.13... I'll fix it up.

Comment (mboersma) [reply pkg/api/k8s_versions.go]: I fixed this--thanks for noticing! `cluster-autoscaler` v1.12.1 appears to be the current 1.12.x release, so I left it there.

Comment (mboersma) [reply pkg/api/k8s_versions.go]: See also #170.

**Pull request merged**